### PR TITLE
Noted Like operator is not supported in .NET Core

### PR DESCRIPTION
**Pull request by rpetrusha**
## Noted Like operator is not supported in .NET Core

//cc @KathleenDollard

### CODE REVIEW

File: docs/visual-basic/language-reference/operators/like-operator.md [hunk 1]
```diff
@@ -25,7 +25,10 @@ ms.assetid: 966283ec-80e2-4294-baa8-c75baff804f9
 ---
 # Like Operator (Visual Basic)
 Compares a string against a pattern.  
-  
+
+> [!IMPORTANT]
+> The `Like` operator is not supported in .NET Core (Microsoft.NET.Sdk) projects.
```
Comment (mairaw) [inline docs/visual-basic/language-reference/operators/like-operator.md]: should we call out .NET Standard too? 

```suggestion
> The `Like` operator is currently not supported in .NET Core and .NET Standard projects.
```

**Review by mairaw — APPROVED** [review]
LGTM. Left a suggestion for you to consider. Plus should we update the date?

**Comment by KathleenDollard** [comment]
Looks great. 

It brings to mind that with VB.NET 16 timeframe, we'll be adding back functionality like this that .NET Core is currently missing. Is there an obvious technique for how we mark features that are missing on .NET Core prior to 16.0?

**Comment by rpetrusha** [comment]
I think we could add a custom label that at least indicates we should review it. What do you think, @mairaw? Possibly just the "updateeachrelease" metadata value, which we can remove once it is supported in .NET Core.

**Comment by mairaw** [comment]
Yeah, that could work. Or we could add a Dev16 project. I think there's more we'll have to worry about for that release.

**Comment by rpetrusha** [comment]
I'll create a Dev16 project.

**Pull request merged**